docs/docs/workflows/terraform-module.md

```diff
@@ -0,0 +1,156 @@
+    name = "constell"
+    location = "northeurope"
+    service_principal_name = "az-sp"
+    resource_group_name = "constell-rg"
+    node_groups = {
+      control_plane_default = {
+        role          = "control-plane"
+        instance_type = "Standard_DC4as_v5"
+        disk_size     = 30
+        disk_type     = "Premium_LRS"
+        initial_count = 2
+      },
+      worker_default = {
+        role          = "worker"
+        instance_type = "Standard_DC4as_v5"
+        disk_size     = 30
+        disk_type     = "Premium_LRS"
+        initial_count = 2
+      }
+    }
+  }
+  ```
+
+  </tabItem>
+
+  <tabItem value="aws" label="AWS">
+
+  ```
+  locals {
+    module_version = "v0.0.0" // insert Constellation version here.
+  }
+
+  module "aws-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/${local.module_version}/terraform-module.zip//modules/aws-constellation"
+    name        = "constell"
+    zone        = "us-east-2c"
+    name_prefix = "example"
+    node_groups = {
+      control_plane_default = {
+        role          = "control-plane"
+        zone          = "us-east-2c"
+        instance_type = "m6a.xlarge"
+        disk_size     = 30
+        disk_type     = "gp3"
+        initial_count = 3
+      },
+      worker_default = {
+        role          = "worker"
+        zone          = "us-east-2c"
+        instance_type = "m6a.xlarge"
+        disk_size     = 30
+        disk_type     = "gp3"
+        initial_count = 2
+      }
+    }
+  }
+  ```
+
+  </tabItem>
+
+  <tabItem value="gcp" label="GCP">
+
+  ```
+  locals {
+    module_version = "v0.0.0" // insert Constellation version here.
+  }
+
+  module "gcp-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/${local.module_version}/terraform-module.zip//modules/gcp-constellation"
+    name    = "constell"
+    project = "constell-proj" // replace with your project id
+    service_account_id = "constid"
+    zone    = "europe-west2-a"
+    node_groups = {
+      control_plane_default = {
+        role          = "control-plane"
+        zone          = "europe-west2-a"
+        instance_type = "n2d-standard-4"
+        disk_size     = 30
+        disk_type     = "pd-ssd"
+        initial_count = 3
+      },
+      worker_default = {
+        role          = "worker"
+        zone          = "europe-west2-a"
+        instance_type = "n2d-standard-4"
+        disk_size     = 30
+        disk_type     = "pd-ssd"
+        initial_count = 2
+      }
+    }
+  }
+  ```
+
+  </tabItem>
+  </tabs>
+
+3. Initialize and apply the module.
+  ```bash
+  terraform init
+  terraform apply
+  ```
+
+## Custom setup
+If you need to separate IAM and cluster management or need custom infrastructure, you can also call the submodules individually.
+You can have a look at the respective convenience module (`{csp}-constellation`) for how the module calls can be structured.
```

Comment:
```suggestion
Look at the respective convenience module (`{csp}-constellation`) for how you can structure the module calls.
```
[prune every excess word](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#be-brief)